sqllogictest-engines/src/external.rs

```diff
@@ -114,7 +118,7 @@ impl AsyncDB for ExternalDriver {
     }
 
     async fn shutdown(&mut self) {
-        self.stdin.shutdown().await.ok();
+        drop(self.stdin.take());
```

Comment:
I just found the implementation of `shutdown` is just `Poll::Ready(Ok(()))`. So the problem is `drop` not called, not because of `shutdown` called.